src/main/scala/beam/agentsim/agents/planning/BeamPlan.scala

```diff
@@ -60,11 +60,11 @@ class BeamPlan extends Plan {
   //////////////////////////////////////////////////////////////////////
   // Beam-Specific methods
   //////////////////////////////////////////////////////////////////////
-  def trips: Vector[Trip] = tours.flatMap(_.trips)
+  lazy val trips: Vector[Trip] = tours.flatMap(_.trips)
```

Comment:
Are we sure that `tours` never changes? If so then maybe it should be implemented that way, otherwise somebody could go and change it...then `trips`, `activities`,... will not reflect accurately.

Reply:
From my understanding yes, it will never change, it's initialized once. I could have rewritten `BeamPlan` in a bit better way without having `var`, but just did quick fix

Reply:
I'd argue that this is an important area - I guess if you create an issue so that it doesn't get forgot then that'd be an acceptable workaround.